contrib/docker/DOCKER.md

```diff
@@ -0,0 +1,99 @@
+# Using Boolector with Docker
+
+Boolector has support for executing Boolector "input files" (e.g., `.smt2`)
+files via the use of a Docker container. The base container used is Alpine
+Linux.
+
+This guide presumes you are familiar with both Docker and Boolector, and are
```

Comment:
I'm not a native speaker, but here is another suggestion (not sure if presume is the right choice here?)

```suggestion
This guide assumes the reader is familiar with both Docker and Boolector, and is
```

Reply:
Done!